syntax = "proto3";

package tesla.proto.energy_device.v1;

option go_package = "github.com/teslamotors/energy_device/pkg/protocol/protobuf/energydevice/v1";
option java_package = "com.tesla.generated.energy_device.v1";
option java_outer_classname = "AuthorizedVerificationType";

enum AuthorizedVerificationType {
  AUTHORIZED_VERIFICATION_TYPE_INVALID         = 0;
  AUTHORIZED_VERIFICATION_TYPE_PRESENCE_PROOF  = 1;
  AUTHORIZED_VERIFICATION_TYPE_BLE             = 2;
  AUTHORIZED_VERIFICATION_TYPE_SIGNED          = 3;
  AUTHORIZED_VERIFICATION_TYPE_HERMES_COMMAND  = 4;
}